SemanticFormsSelect.hooks.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+class SemanticFormsSelectHooks {
```

Comment:
Why do I need such class here and especially out of the normal PSR-4 / src directory.

I __did not clean-up__ this extension to see stuff like this (which is common practice in WMF related development) to be floating around.